docs/sources/datasources/google-cloud-monitoring/_index.md

```diff
@@ -217,10 +206,10 @@ The SLO query builder in the Google Cloud Monitoring data source allows you to d
 To create an SLO query, follow these steps:
 
 1. Choose the option **Service Level Objectives (SLO)** in the **Query Type** dropdown.
-1. Choose a project from the **Project** dropdown.
-1. Choose an [SLO service](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/services) from the **Service** dropdown.
-1. Choose an [SLO](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/services.serviceLevelObjectives) from the **SLO** dropdown.
-1. Choose a [time series selector](https://cloud.google.com/monitoring/service-monitoring/timeseries-selectors#ts-selector-list) from the **Selector** dropdown.
+2. Choose a project from the **Project** dropdown.
```

Comment:
These should all be "1.". Please update to match correct format.

Reply:
My IDE fights me 🙃 Is there a reason they should all be 1?
Is that blocking? I can work around it if it is absolutely necessary. Let me know. 🙂

Reply:
Unfortunately, yes. They all need to be "1". Thanks for checking!

Reply:
Done!